events_api.go

```diff
@@ -178,18 +197,30 @@ func (a *EventsAPIServer) handleMessageEvent(ev *slackevents.MessageEvent) {
 				Text:            ev.Text,
 				Timestamp:       ev.TimeStamp,
 				ThreadTimestamp: ev.ThreadTimeStamp,
-				Edited: &slack.Edited{
-					User:      ev.Edited.User,
-					Timestamp: ev.Edited.TimeStamp,
-				},
-				SubType:        ev.SubType,
-				EventTimestamp: ev.EventTimeStamp.String(),
-				BotID:          ev.BotID,
-				Username:       ev.Username,
-				Icons: &slack.Icon{
-					IconURL:   ev.Icons.IconURL,
-					IconEmoji: ev.Icons.IconEmoji,
-				},
+				Edited:          edited,
+				SubType:         ev.SubType,
+				EventTimestamp:  ev.EventTimeStamp.String(),
+				BotID:           ev.BotID,
+				Username:        ev.Username,
+				Icons:           icons,
+			},
+		},
+	}
+}
+
+func (a *EventsAPIServer) handleAppMentionEvent(ev *slackevents.AppMentionEvent) {
+	a.events <- slackEvent{
```

Comment:
Here you are pushing a new event type to the events channel but seems you forgot to handle the `slackevents.AppMentionEvent` type in the adapters event loop? I.e. without the actual handler, this will either be ignored anyway or be logged as unknown type.

Reply:
Are you sure about that? It seems to be handled the using the same pattern as ReactionAdded/Message events are?

Reply:
Yeah my bad, I must have missed it somehow